go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/checkpoint-restore/go-criu/v7
 
-go 1.18
+go 1.20
```

Comment:
@Luap99 Would you be able to also update the CI workflow in [main.yml](https://github.com/checkpoint-restore/go-criu/blob/ae18f4225e2e1107609640a7d50242eaa4b41536/.github/workflows/main.yml#L16) with this change?

Reply:
sure